base/c.jl

```diff
@@ -296,34 +294,40 @@ function transcode(::Type{UInt16}, src::AbstractVector{UInt8})
     dst = UInt16[]
     i, n = 1, length(src)
     n > 0 || return dst
-    sizehint!(dst, 2n)
+    sizehint!(dst, 2 * n)
     a = src[1]
     while true
         if i < n && -64 <= a % Int8 <= -12 # multi-byte character
-            b = src[i += 1]
+            b = src[i+=1]
```

Comment:
wait, why do other operators get spaces, but `+=` isn't allowed them here?

Reply:
afaik most of the time things inside `foo[...]` have no spaces.